policy/modules/system/init.te

```diff
@@ -618,6 +620,10 @@ ifdef(`init_systemd',`
 		unconfined_create_keys(init_t)
 		unconfined_write_keys(init_t)
 	')
+
+	optional_policy(`
+		virt_stream_connect(init_t)
```

Comment:
This is related to machined?

Reply:
From what I can tell, it's related to `virtlogd-admin.socket`, so virtlogd needs a socket activation call instead.